hawkbit-rest/hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtRolloutResource.java

```diff
@@ -103,26 +103,17 @@ public ResponseEntity<PagedList<MgmtRolloutResponseBody>> getRollouts(
         final boolean isFullMode = parseRepresentationMode(representationModeParam) == MgmtRepresentationMode.FULL;
 
         final Pageable pageable = new OffsetBasedPageRequest(sanitizedOffsetParam, sanitizedLimitParam, sorting);
-        final Slice<Rollout> rollouts;
         final long totalElements;
```

Comment:
Total elements should be declared with initialisation. E.g.:
final long totalElements = rollouts.getTotalElements();

Reply:
And please add comments to this PR